src/libraries/System.IO.Pipes.AccessControl/ref/System.IO.Pipes.AccessControl.csproj

```diff
@@ -3,12 +3,10 @@
     <TargetFrameworks>net5.0;netstandard2.0;net461</TargetFrameworks>
     <Nullable>enable</Nullable>
   </PropertyGroup>
-  <!-- DesignTimeBuild requires all the TargetFramework Derived Properties to not be present in the first property group. -->
-  <PropertyGroup>
-    <IsPartialFacadeAssembly Condition="'$(TargetFramework)' == 'net461'">true</IsPartialFacadeAssembly>
-  </PropertyGroup>
   <ItemGroup>
-    <Compile Include="System.IO.Pipes.AccessControl.cs" />
+    <Compile Include="System.IO.Pipes.AccessControl.PipesAclExtensions.cs" />
```

Comment:
Ok seems this is another **partial** facade.

Reply:
Yes, System.Threading.AccessControl as well.